person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
```

Comment:
### Optional
- [ ] It would be nice if we declare the attr_reader and attr_accessor before the initialize method. It is best practice to declare them before creating the initialize method